OWNERS

```diff
@@ -2,3 +2,7 @@ approvers:
 - deads2k
 - jnpacker
 - qiujian16
+# Temporary approvers added for Konflux migration to facilitate release repo prow configuration updates
```

Comment:
This will make the upstream and downstream diverge. And the "sync fork" from upstream will fail.